test/bpf/verifier-test.sh

```diff
@@ -109,17 +107,8 @@ function cg_prog_type_init {
 
 	mkdir -p ${TESTMOUNT}
 
-	# These show up in sockops progs; skip them quietly.
-	prog_types["int"]="SKIP"
-	prog_types["_version"]="SKIP"
-	prog_types["version"]="SKIP"
 	if $BPFTOOL help 2>&1 | grep -q feature; then
-		if check_macro SOCK_OPS; then
-			prog_types["sockops"]="sock_ops"
-			attach_types["sockops"]="sock_ops"
-		fi
 		if check_macro SK_MSG; then
-			prog_types["sk_msg"]="sock_ops"
 			attach_types["sk_msg"]="msg_verdict"
 		fi
```

Comment:
This entire block should be removed AFAICT. We're no longer using `sk_msg` BPF programs after this PR.